taskipy/task_runner.py

```diff
@@ -22,6 +25,14 @@ def run(self, task_name: str, args: List[str]) -> int:
                 command, shell=True, cwd=Path(self.__pyproject_path).parent
             )
 
+            def stop_handler(signum: int, frame) -> None:  # pylint: disable=unused-argument
```

Comment:
Maybe we should explain what this function actually does? how about renaming into `send_signal_to_task_process` :)?